### PR TITLE
Base64Decode Script

### PR DESCRIPTION
**Pull request by compsecmonkey**
## Status
Ready

## Description
Adds a Base64Decode script to Demisto

## Required version of Demisto
Any

## Does it break backward compatibility?
   - No

## Additional changes
None

### CODE REVIEW

File: Scripts/script-Base64Decode.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          "Contents": "String '{}' is an invalid base64 encoded string".format(input_string)
+      })
+type: python
+tags: []
+comment: Decode a base64 encoded string
+enabled: true
+args:
+- name: input
+  required: true
+  default: true
```
Comment (Itay4) [inline Scripts/script-Base64Decode.yml]: Not sure why you set the default value for `true`, but anyway I think there shouldn't be any default for this argument.

Comment (compsecmonkey) [reply Scripts/script-Base64Decode.yml]: That is not setting the default value to "true" but marking the field as a default field. This was modeled after the already exsiting Base64Encode script.

**Comment by compsecmonkey** [comment]
Good call on the EntryContext. That was overlooked and is now fixed.

File: Scripts/script-Base64Decode.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  try:
+      decoded = b64decode(input_string)
+      demisto.results({
+      "ContentsFormat": formats["text"],
+      "Type": entryTypes["note"],
+      "Contents": decoded,
+      "EntryContext": {"Base64.decoded": decoded}
+      })
+
+  except TypeError:
```
Comment (Itay4) [inline Scripts/script-Base64Decode.yml]: Please handle `UnicodeEncodeError` exceptions as well.

File: Scripts/script-CommonServerPython.yml [hunk 3]
```diff
@@ -531,12 +531,8 @@ script: |-
       elif isinstance(data, list):
           string_list = []
           for d in data:
-              try:
-                  string_list.append(str(d))
-              except UnicodeEncodeError:
-                  string_list.append(d.encode('utf-8'))
-
-          return ',\n'.join(string_list)
+              string_list.append(str(d) if not isinstance(d, STRING_TYPES) else d)
```
Comment (Itay4) [inline Scripts/script-CommonServerPython.yml]: Not sure why you changed that, there is no call to `flattenCell` in the script

Comment (anara123) [reply Scripts/script-CommonServerPython.yml]: @compsecmonkey any update here?

**Pull request closed without merging**